Sources/Private/CoreAnimation/Animations/VisibilityAnimation.swift

```diff
@@ -28,7 +28,11 @@ extension CALayer {
     animation.keyTimes = [
       NSNumber(value: 0.0),
       NSNumber(value: max(Double(context.progressTime(for: inFrame)), 0)),
-      NSNumber(value: min(Double(context.progressTime(for: outFrame)), 1)),
+      // Anything visible during the last frame should stay visible until the absolute end of the animation.
+      //  - This matches the behavior of the main thread rendering engine.
+      context.timeRemapping(outFrame) == context.animation.endFrame
+        ? NSNumber(value: Double(1.0))
```

Comment:
Without this, some animations (like `LottieLogo1.json`) would unexpectedly render a blank frame as the last frame. 

This matches the behavior of the main thread rendering engine as far as I can tell, but causes bugs. e.g. this change is why the "1" and "9/10" overlap in this screenshot:

<img src="https://github.com/airbnb/lottie-ios/assets/1811727/d65617a2-ddf2-43f7-bc43-8b49f5d09814" width=350>

For now let's prioritize parity with the main thread rendering engine.